change/@fluentui-react-components-dc40066c-1a06-4bad-aca4-f52297989324.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Component: Adds TeachingBubblePreview Components",
```

Comment:
preview packages are not added to suite anymore. https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-v9-Component-Roadmap#%EF%B8%8F-previewunstable